cdap-kubernetes/src/main/java/io/cdap/cdap/master/environment/k8s/KubeMasterEnvironment.java

```diff
@@ -388,8 +384,8 @@ public Supplier<TwillRunnerService> getTwillRunnerSupplier() {
   }
 
   @Override
-  public Optional<MasterEnvironmentTask> getTask() {
-    return Optional.ofNullable(podKillerTask);
+  public Collection<MasterEnvironmentTask> getTasks() {
+    return SetUtils.hashSet(podKillerTask, kubeJobCleaner);
```

Comment:
No need to use apache common lib in this case. Just return `Arrays.asList(podKillerTask, kubeJobCleaner)`.